src/components/SharedComponents/Sheets/WarningSheet.tsx

```diff
@@ -35,13 +36,14 @@ const WarningSheet = ( {
   testID,
   text
 }: Props ) => {
-  const buttons = [
+  const buttons: ButtonConfiguration[] = [
     {
       title: buttonText,
       onPress: confirm,
       disabled: loading,
       level: buttonType || "warning",
       loading,
+      isPrimary: false,
```

Comment:
`isPrimary` is a required property of the configuration. It was previously implicitly `undefined` false-y, and now it's actually specified.